<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Compress</class>
 <widget class="QDialog" name="Compress">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>591</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>生成压缩文件</string>
   </property>
   <widget class="QPushButton" name="StartCompress">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>开始压缩</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FilePathBTN">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>浏览...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditFileName">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>保存路径：</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>591</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>解压缩文件</string>
   </property>
   <widget class="QPushButton" name="StartUncompress">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>开始解压</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FilePathBTN_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>70</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>浏览...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>保存路径：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditFileName_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="EditFilePath_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>401</width>
      <height>20</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="FileNameBTN_2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>71</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>浏览...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>压缩文件：</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="close">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>370</y>
     <width>100</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>601</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
